test/Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests/ResponseHeaderTests.cs

```diff
@@ -0,0 +1,148 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+using SameSiteMode = System.Web.SameSiteMode;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests;
+
+[Collection(nameof(SelfHostedTests))]
+public class ResponseHeaderTests
+{
+    private readonly string ContentValue = Guid.NewGuid().ToString();
+
+    [Fact]
+    public async Task SetCookie()
+    {
+        var result = await RunAsync(context =>
```

Comment:
nit: httpresponsemessage should have a `using`; not sure why there's no analyzer catching that

Reply:
@twsouthwick good catch. I just updated the PR and added `using`. I can update the analyzer to catch this type of errors in a separate PR.